iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/extensions/SlackTerminalImpl.java

```diff
@@ -46,7 +46,7 @@ public SlackTerminal setTerminal(Terminal terminal) {
     }
 
     @Override
-    public boolean isCleanable() {
+    public boolean isEmpty() {
```

Comment:
I think this method should be define on the SlackTerminal API, not for the all extensions (that could concern everything in powsybl, not necessarily IIDM extensions)

Reply:
Yes (like it was before that PR), if the serializer is responsible of checking if the serialization is needed, there's no need for that in all the extesions. And what about the name btw, isCleanable / isEmpty?